src/sentry/integrations/vercel/integration.py

```diff
@@ -115,13 +127,28 @@ def build_integration(self, state):
             user = client.get_user()
             name = user["name"]
 
+        try:
+            webhook = client.create_deploy_webhook()
+        except ApiError as err:
+            logger.info(
+                "vercel.create_webhook.failed",
+                extra={"error": six.text_type(err), "external_id": external_id},
+            )
+            try:
+                details = err.json["messages"][0].values().pop()
+            except Exception:
+                details = ""
```

Comment:
@MeredithAnya Nit: maybe in this case we should say something like `unknown error`